app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala

```diff
@@ -36,7 +36,9 @@ class FinancialDataConnector @Inject() (config: AppConfig, implicit val httpClie
   override protected val logger: Logger = Logger(this.getClass)
 
   def getFinancialData(
-    appaId: String
+    appaId: String,
+    open: Boolean = true,
```

Comment:
Rename this to onlyOpenItems through the code please. Also shouldn't default to any value

Reply:
done